src/reconcile/rec_write.c

```diff
@@ -2500,7 +2507,10 @@ __rec_write_wrapup(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_PAGE *page)
          */
         ref = r->ref;
         if (__wt_ref_is_root(ref)) {
-            __wt_checkpoint_tree_reconcile_update(session, &ta);
+            if (__wt_process.page_stats_2022)
```

Comment:
Can you add a comment to describe why we are setting the count to zero

Reply:
I thought the checkpoint information was not persisting correctly in the scenario of an empty table, but seems like it might be because we're not initialising empty tables properly in the reconciliation flow. I'll need to look at this again to confirm.

Reply:
I think the checkpoint information is different to the information that is packed onto the disk. Here I think we should set to zero since we are checkpointing an empty page. Currently, it seems that the checkpoint information is correct though I have doubts the zero is being packed to disk correctly.